src/read.jl

```diff
@@ -192,6 +194,21 @@ function read_demand_flexibility(filepath, interval)::DemandFlexibility
             )
             demand_flexibility["duration"] = interval
         end
+        
+        # Try reading the cost for up- and down-shifting loads
+        for cost_file_postfix in ("up", "dn")
+            try 
+                demand_flexibility["cost_$cost_file_postfix"] = CSV.File(
+                    joinpath(filepath, "demand_flexibility_cost_$cost_file_postfix.csv")
+                ) |> DataFrames.DataFrame
+                println("...loading demand flexibility $cost_file_postfix-shift cost profiles")
+            catch e
+                println(
+                    "Demand flexibility down-shift cost profiles not found in " * filepath
+                )
+                demand_flexibility["cost_$cost_file_postfix"] = nothing
```

Comment:
What does `cost_file_postfix` mean?

Reply:
> What does `cost_file_postfix` mean?

Oh I meant "suffix".. the conterpart is "prefix". I need to review my english,

Reply:
Aha! Don't worry, English does not make any sense.

Reply:
> Aha! Don't worry, English does not make any sense.

I have fixed locally. I will wait for the other comments and update later.

Reply:
The code looks good to me, and @lanesmith suggested in the old PR that once his comments are addressed (which it seem like they are now) that it's ready to go. If you push the small changes requested on this one, I will approve and we can get this one merged, and then we will rebase #145 and merge that one as well.

Reply:
> The code looks good to me, and @lanesmith suggested in the old PR that once his comments are addressed (which it seem like they are now) that it's ready to go. If you push the small changes requested on this one, I will approve and we can get this one merged, and then we will rebase #145 and merge that one as well.

I just pushed through interactive rebase.. why there are the strange log texts in the source code?

Reply:
@dongqi-wu I think you need to delete those texts manually when resolving conflicts. Those are the markups that Github automatically generated when conflicts are detected, which tells you what are the conflicts explicitly.

Reply:
It looks like there were some merge conflicts that got committed. Would you like me to force-push a fixed commit with a better commit message into your repo?

Reply:
> It looks like there were some merge conflicts that got committed before being resolved. Would you like me to force-push a fixed commit with a better commit message into your repo?

Hi Daniel I just saw the message.. I was trying to fix myself and it seems there are some problems with the commits (as of now). The changes seem to be separated into the 2 commits and when I do interactive rebase and drop the 1st one it only keep the change in the 2nd one. I think I am supposed to make 1 commit that is the union the current 2 commits. Is there something I did wrong?

Reply:
You didn't want to drop, you wanted to designate the second commit as a `fixup`, which will combine that commit's changes with the commit before. Next time you have a git issue, send us a message on Slack, that should be the fastest way to resolve it.